crates/driver/src/tests/setup/mod.rs

```diff
@@ -1307,8 +1250,13 @@ impl<'a> Settle<'a> {
 
     /// Expect the /settle endpoint to return a 400 BAD REQUEST response.
     pub fn err(self) -> SettleErr {
```

Comment:
I know this is not your change, but why this is not `Option<SettleErr>` so it is consistent with Rust std function `.err()` (and the panic is handled by the caller)? having two kind of `.err()` outcomes could be confusing :thinking:

Reply:
This code is used in tests only.